conf/gunicorn-dev.conf.py

```diff
@@ -0,0 +1,5 @@
+bind = "0.0.0.0:8001"
+reload = True
+reload_extra_files = "lms/templates"
+timeout = 0
+workers = 2
```

Comment:
There's a significant difference between `1` worker and `>1` workers, so I think we probably want to set this to `2` in dev.